README.md

```diff
@@ -77,6 +77,14 @@ export LIBVA_DRIVERS_PATH=<path-contains-iHD_drv_video.so>
 export LIBVA_DRIVER_NAME=iHD
 ```
 
+## Docker Image
+Docker images are fully qualified by media stack suites. Latest image is targeting to Ubuntu 18.04. You can get it by
+```
+$ docker pull intelmediadriver/media-driver
+```
+Other images you can check it through [hub docker](https://hub.docker.com/r/intelmediadriver/media-driver/tags/)
```

Comment:
How these images are related to #327? are they related?

Reply:
Yes, they are related. Once the PR merged, the docker image will be triggered automatically. After that, once the dockerfile updated, like for weekly/bi-weekly tag update, image will be auto-triggered too to make sure user always get latest tested config.